core/src/main/web/app/mainapp/components/notebook/cell-directive.js

```diff
@@ -217,6 +217,26 @@
         $scope.isCodeCell = function() {
           return $scope.cellmodel.type == 'code';
         };
+
+        $scope.evaluateSection = function($evt) {
```

Comment:
we already have this feature, no need to reimplement
https://github.com/twosigma/beaker-notebook/blob/a811e079ac5afc8ac107ea6dffc3c565509f00d5/core/src/main/web/app/mainapp/components/notebook/sectioncell-directive.js#L130